src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -211,6 +212,11 @@ public GhprbTrigger(String adminlist,
         this.permitAll = permitAll;
         this.autoCloseFailedPullRequests = autoCloseFailedPullRequests;
         this.displayBuildErrorsOnDownstreamBuilds = displayBuildErrorsOnDownstreamBuilds;
+
+        if (commentFilePath != null) {
+          extensions.add(new GhprbCommentFile(commentFilePath));
```

Comment:
Stylistic request.  Can you indent this with four spaces instead of tow like the rest of the file?

Reply:
Done, sorry for the delay.